utils.js

```diff
@@ -488,3 +488,31 @@ export const getOptionsFromHash = (hash, label = 'name') => {
 
   return results
 }
+
+export const getPrice = (() => {
```

Comment:
Currently `getPrice.min(p) getPrice.max(p)` returns string and meant to use always together, so it can be simplified to `getPrice(p)`